src/aleph/handlers/content/vm.py

```diff
@@ -90,6 +96,7 @@ def _map_content_to_db_model(item_hash: str, content: ProgramContent) -> Program
 
 def _map_content_to_db_model(item_hash, content):
     db_cls = ProgramDb if isinstance(content, ProgramContent) else VmInstanceDb
+    payment_type = PaymentType.hold
```

Comment:
You probably want to move this line next to line 127 so all the code related to this variable is grouped together